src/lib/components/Molecule3dViewer.js

```diff
@@ -32,12 +32,19 @@ export default class Molecule3dViewer extends Component {
     }
 
     render() {
-        const {id} = this.props;
+        const {id, selectionType} = this.props;
+
+        // molecule-3d-for-react requires the first letter in the selection type to be
```

Comment:
```suggestion
        // molecule-3d-for-react requires the selection type to be
```
Capitalizing a word means precisely writing its first letter uppercase.